internal/data/character.go

```diff
@@ -45,30 +47,14 @@ type Character struct {
 	Meseta            uint32
 }
 
-// FindCharacter returns the Character stored in the specified slot for the account
-// or none if no Character exists.
-func FindCharacter(account *Account, slotNum int) (*Character, error) {
```

Comment:
My thinking for these files was to keep everything grouped by the entities on which they operate. For instance if you want to perform a CRUD operation on a particular object, you look in`data/<object>.go` where you'll find all of the methods that accept or return instances of that object. In this case, `character.go` no longer has methods to retrieve characters so there's now a bit of conflation going on in that there's implicit knowledge/context needed in order to figure out how to query for a character.

It's not a big deal for now, but I'm betting this change is going to need to be backed out later as the Block server might not always have the `account` reference needed for Character operations.

Reply:
I thought so, but in previous state FindCharacter was practically the same FindCharacterInSlot but with a different name. I'm also more of a fan of logical grouping for such kind of logic so moving it to character.go would be my suggestion. Though in this case it will lose some of it's "beauty" with being a struct mehod with account.FindOnMe - so I really like it better this way at least for now. What do you propose?

Reply:
Ah I see, I had forgotten about `(a *Account) FindCharacterInSlot`; that's definitely redundant if `FindCharacter(account, slot)` exists. Let's keep it consistent with the latter since the former is the exception to the convention and we can just remove `(a *Account) FindCharacterInSlot`. We can revisit later if we find it's annoying. WDYT?

Reply:
Well I personally like (a *Account) FindCharacterInSlot better, but for consistency reasons I can move it to character go as the  FindCharacter(account, slot). I've also decided to remove the latter because it was only called once while account.FindCharacterInSlot had few different usages. No hassle to change though

Reply:
Done